generators/chipyard/src/main/scala/upf/ChipTopUPF.scala

```diff
@@ -0,0 +1,72 @@
+// See LICENSE for license details
+package chipyard.upf
+
+import chipyard.TestHarness
+import chipyard.{ChipTop, DigitalTop}
+import freechips.rocketchip.diplomacy.LazyModule
+import chisel.{UPFAspect, UPFFunction, ChiselUPFElement}
+
+import scala.collection.mutable.ListBuffer
+
+object ChipTopUPF {
+  def default: UPFFunction = {
+    case top: ChipTop => {
```

Comment:
To do the proper matching, you should copy what they previously did: https://github.com/ucb-bar/chipyard/blob/a11263d1b17dbac65130aff41180404dbe712baf/generators/chipyard/src/main/scala/ChipTop.scala#L25